### DISCUSSION TITLE
Ocr 2.0 Release（拖延症） | 连波的闲谈杂鱼

### DISCUSSION THREAD

**Discussion by giscus** [godruoyi/gblog#37] [Announcements]
# Ocr 2.0 Release（拖延症） | 连波的闲谈杂鱼

三年前在 Github 发布了一个图片识别的 PHP 扩展包 godruoyi/ocr，主要功能是整合几大厂商的图片识别接口，方便统一调用。

https://godruoyi.com/posts/procrastination/

**Comment by liutiedong** [comment]
hi

**Reply by godruoyi** [reply]
hi @liutiedong, welcome to my blog.
